src/main/java/rs/teslaris/core/importer/model/oaipmh/organisationunit/package-info.java

```diff
@@ -0,0 +1,13 @@
+@XmlSchema(
+    namespace = "http://www.openarchives.org/OAI/2.0/",
+    elementFormDefault = XmlNsForm.QUALIFIED,
+    xmlns = {
+        @jakarta.xml.bind.annotation.XmlNs(prefix = "xsi", namespaceURI = "http://www.w3.org/2001/XMLSchema-instance"),
+        @jakarta.xml.bind.annotation.XmlNs(prefix = "xml", namespaceURI = "http://www.w3.org/XML/1998/namespace"),
+        @jakarta.xml.bind.annotation.XmlNs(prefix = "xs", namespaceURI = "http://www.w3.org/2001/XMLSchema")}
+)
+
+package rs.teslaris.core.importer.model.oaipmh.organisationunit;
+
+import jakarta.xml.bind.annotation.XmlNsForm;
+import jakarta.xml.bind.annotation.XmlSchema;
```

Comment:
It is not possible to have just one package-info.java for all?

Reply:
Unfortunately no, jaxb requires it this way. We had same problem to other project during our graduate studies while using jaxb. I will also write this down and research it, if I find any workaround I will suggest it.